data-manipulation/compression/create-cabinet-file.yml

```diff
@@ -0,0 +1,23 @@
+rule:
+  meta:
+    name: create Cabinet file
```

Comment:
Thoughts on changing this to `create Cabinet on Windows` to make it more clear?

Reply:
Thanks! Renamed the rule in https://github.com/mandiant/capa-rules/pull/808/commits/ac09516cca968e54d6b5764405337cba3f290c21